### ISSUE TITLE
Shell Help Conflict

### ISSUE THREAD

**Issue by Econ441** [h3rald/min#127]
```
$ ?cons
=== cons [symbol]
a1 (a*) ==> (a1 a*)

  Prepends a1 to the quotation on top of the stack.
 (
   1
   4
   5
  )
```

The needless part of symbol documentation is that we see current-stock `( 1 4 5 )`
I think this is a bug. People may conflict with current-stock. The stock seems belong to documentation of symbol.

**Comment by h3rald** [comment]
Yeah, it is not nice. 

But at the same time I cannot automatically clear the stack whenever you Issue the help command, so I think I'll just add a `===` on the last line of the help, as a separator.

**Comment by Econ441** [comment]
> Yeah, it is not nice.
> 
> But at the same time I cannot automatically clear the stack whenever you Issue the help command, so I think I'll just add a `===` on the last line of the help, as a separator.

But documentation may include a seperator which is `===` . Doesn't this make conflict?

**Comment by h3rald** [comment]
Well, no, there shouldn't be any need to put === at the beginning of 
 the last line, on its own... and even if you do, the description is automatically indented with two spaces so no, there is no conflict...

**Comment by h3rald** [comment]
Fixed in v0.32.0.
